profiles/base.yaml

```diff
@@ -537,6 +537,9 @@ safe_room_tip_amount:
 safe_room_empath:
 safe_room_empaths:
 
+# override the room you go to when hunting-buddy ends
+force_after_hunt_room:
```

Comment:
💡 We also have a "prehunt_buffing_room" setting, thoughts if this new setting were named "posthunt_safe_room"?

I see why you would have chosen "force", and I suppose to me the force variables tend to be isolated to overriding the hometown.